src/main/kotlin/io/github/inductiveautomation/kindling/utils/TabStrip.kt

```diff
@@ -74,6 +93,13 @@ open class TabStrip : DnDTabbedPane() {
                             }
                         },
                     )
+                    if (tabsEditable && isTabClosable(tabIndex)) {
```

Comment:
Minor: Could move `val closable` from line 103 above this check, so `if (tabsEditable && closable)`